packages/utils/smash.js

```diff
@@ -5,17 +5,17 @@
  * @returns { p: { a: 1, b: 2, c: { d: 1, e: 2 } } }
  */
 function main(args) {
-    let parent = args['$fieldName']
+    let parent = args['$fieldName'];
```

Comment:
I can update combinators as well.  Should we add jshint to our travis code scan?